src/components/Checkbox.tsx

```diff
@@ -3,36 +3,82 @@ import { colors } from "../theme";
+export type CheckboxSize = 1.4 | 1.6 | 1.8 | 2;
 
 export const checkboxVariants = {
   primary: {
-    accentColor: colors.palette.mediumBlue,
-    boxShadow: 'none',
+    backgroundColor: colors.palette.mediumBlue,
+    color: 'inherit',
+    unCheckedBorder: `1px solid ${colors.palette.neutralThin}`,
+    checkedBorder: `1px solid ${colors.palette.mediumBlue}`,
+    backgroundImage: 'data:image/svg+xml,<svg height="125px" width="125px" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 17.837 17.837" xml:space="preserve" fill="%23000000"><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g><g id="SVGRepo_iconCarrier"><g><path style="fill:%23fff;" d="M16.145,2.571c-0.272-0.273-0.718-0.273-0.99,0L6.92,10.804l-4.241-4.27 c-0.272-0.274-0.715-0.274-0.989,0L0.204,8.019c-0.272,0.271-0.272,0.717,0,0.99l6.217,6.258c0.272,0.271,0.715,0.271,0.99,0 L17.63,5.047c0.276-0.273,0.276-0.72,0-0.994L16.145,2.571z"></path></g></g></svg>'
   },
   light: {
-    accentColor: colors.palette.white,
-    boxShadow: '0 0 1px 0',
+    backgroundColor: colors.palette.white,
+    color: 'inherit',
+    unCheckedBorder: `1px solid ${colors.palette.pale}`,
+    checkedBorder: `1px solid ${colors.palette.pale}`,
+    backgroundImage: 'data:image/svg+xml,<svg height="125px" width="125px" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 17.837 17.837" xml:space="preserve" fill="%23000000"><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g><g id="SVGRepo_iconCarrier"><g><path style="fill:%235e5e5e;" d="M16.145,2.571c-0.272-0.273-0.718-0.273-0.99,0L6.92,10.804l-4.241-4.27 c-0.272-0.274-0.715-0.274-0.989,0L0.204,8.019c-0.272,0.271-0.272,0.717,0,0.99l6.217,6.258c0.272,0.271,0.715,0.271,0.99,0 L17.63,5.047c0.276-0.273,0.276-0.72,0-0.994L16.145,2.571z"></path></g></g></svg>'
+  },
+  error: {
+    backgroundColor: colors.palette.paleRed,
+    color: colors.palette.darkRed,
+    unCheckedBorder: `1px solid ${colors.palette.lightRed}`,
+    checkedBorder: `1px solid ${colors.palette.lightRed}`,
+    backgroundImage: 'data:image/svg+xml,<svg height="125px" width="125px" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 17.837 17.837" xml:space="preserve" fill="%23000000"><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g><g id="SVGRepo_iconCarrier"><g><path style="fill:%23C22032;" d="M16.145,2.571c-0.272-0.273-0.718-0.273-0.99,0L6.92,10.804l-4.241-4.27 c-0.272-0.274-0.715-0.274-0.989,0L0.204,8.019c-0.272,0.271-0.272,0.717,0,0.99l6.217,6.258c0.272,0.271,0.715,0.271,0.99,0 L17.63,5.047c0.276-0.273,0.276-0.72,0-0.994L16.145,2.571z"></path></g></g></svg>'
+  },
```

Comment:
would you mind creating an icons file somewhere (unless it already exists) and importing the svgs as named icons